lib/setup.js

```diff
@@ -202,6 +203,32 @@ export default function (app) {
   // Sets up mobile marketing signup modal
   app.use(marketingSignupModal)
 
+  // Setup hot-swap loader
+  if (NODE_ENV === 'development') {
+    const reloadAndMount = createReloadable(app, require)
```

Comment:
This is due to how modules work in Node, and scope; from the perspective of each library, each has its own `require` context and cache and when a file changes it needs to know where to look to find and remove it. In this sense, while `require` is available throughout the entirely of a Node app it's not exactly a global.